integration-cli/docker_cli_inspect_test.go

```diff
@@ -42,6 +42,12 @@ func (s *DockerCLIInspectSuite) TestInspectImage(c *testing.T) {
 	// fails, fix the difference in the image serialization instead of
 	// updating this hash.
 	imageTestID := "sha256:11f64303f0f7ffdc71f001788132bca5346831939a956e3e975c93267d89a16d"
+	usesContainerdSnapshotter := false // TODO(vvoland): Check for feature flag
```

Comment:
This is still disabled by default for now, but will be enabled in a future update